ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+************ STOP!! ************
+If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
+the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+-->
+
+<!--
+- For *SUPPORT QUESTIONS*, use the
+[Traffic Control slack channels](https://traffic-control-cdn.slack.com) or [Traffic Control mailing lists](http://trafficcontrol.apache.org/mailing_lists/).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR. -->
+
+**I'm submitting a ...**
```

Comment:
These should be section headings, starting with one pound sign (<kbd>#</kbd>) per heading level. Presumably the issue name itself will be the top-level heading, so I'd suggest using 2.